client/src/Components/List.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react'
+import List from '@mui/material/List'
+import ListItemText from '@mui/material/ListItemText';
+import ListItemButton from '@mui/material/ListItemButton';
+import AboutButton from './AboutButton'
+import { Box, Typography, Drawer} from '@mui/material'
+import AppBar from '@mui/material/AppBar';
+import Page from './Page';
+
+
+export default function ListView() {
+    const stops = ['Main Entrance', 'Lower Campus', 'Village/Farm',
+        'East Remote', 'East Field/OPERS', 'Cowell', 'Crown', '9/10', 'Science Hill',
+        'Kresge', 'Kerr Hall', 'RCC/Porter', 'Family Student Housing', 'Oakes/West Remote', 'Arboretum'];
+    //const [showPage, setShowPage] = useState('');
+    const [isDrawerOpen, setDrawerOpen] = useState(false)
+    const [stop,displayStop] = useState('')
+    const handleDrawerOpen = () => {
+        setDrawerOpen(true)
+      }
+    
+      const handleDrawerClose = () => {
+        setDrawerOpen(false)
+      }
+    const setStop = (newStop) => {
+        displayStop(newStop)
+    }
+    return (
+        <>
+            <Box sx={{
+                display:'flex',
+                height: window.innerHeight,
+                width: '100vw',
+                flexDirection: 'column',
+                justifyContent: 'flex-start',
+                paddingTop: '5vh',
+                alignItems: 'center',
+                backgroundColor: 'background.default'
+            }}
+            
+            >
+
+                <List sx={{
+                    width: '100%',
+                    position: 'absolute',
+                    left: '100px',
+                    top: '80px',
+                    overflow: 'auto',
+                    maxHeight: '100%',
+                    backgroundColor: 'background'
+                }}>
+                    {stops.map((stop) => (
+                        <ListItemButton onClick ={()=> {handleDrawerOpen(); setStop(stop)}}>
```

Comment:
These buttons still appear on the middle of the screen and extend all the way to the right, try to center this somehow.

Also, try not to hardcode absolute position anything thats is not the outermost container, in this case box. 

since the box is already a flexbox thats centered, you can remove the position absolute, left, and top